pylarky/starlarky.py

```diff
@@ -0,0 +1,30 @@
+import tempfile
+import subprocess
+
+RUNNER_EXECUATBLE = 'larky-runner'
+INPUT_PARAM = '-input'
+OUTPUT_PARAM = '-output'
+
+
+def evaluate(script, input_data: str) -> str:
+    output_file = tempfile.NamedTemporaryFile(mode='w+')
+    input_file = tempfile.NamedTemporaryFile(mode='w+')
+    input_file.write(input_data)
+    input_file.flush()
+
+    __evaluate(script, input_file.name, output_file.name)
+
+    return open(output_file.name, mode='r').read()
+
+
+def __evaluate(script, input_path, output_path):
+    code = subprocess.call([RUNNER_EXECUATBLE,
+                            INPUT_PARAM, input_path,
+                            OUTPUT_PARAM, output_path,
+                            script])
+    if code != 0:
+        raise FailedEvaluation(f'Evaluation returned non-zero return code. Code: {code}')
```

Comment:
IMO exit code gives not much info about an error. Please consider reading stderr (or stdout if needed) of larky-runner and inject it into the exception message.